src/utils/TestUser.ts

```diff
@@ -0,0 +1,148 @@
+import testAvatar from "../assets/images/testAvatar.png";
+import testAvatar2 from "../assets/images/testAvatar2.png";
+
+export interface CvBase {
+  avatar: string;
+  name: string;
+  githubLink: string;
+  tel: string;
+  email: string;
+  description: string;
+  githubUsername: string;
+}
+
+export interface CvGrades {
+  courseCompletion: number;
+  courseEngagment: number;
+  projectDegree: number;
+  teamProjectDegree: number;
+}
+
+export interface CvExpectations {
+  workPlacePreffer: string;
+  prefferCity: string;
+  expectedContractType: string;
+  expectedSalary: string;
+  consentForFreeInternships: string;
+  commercialProgrammingExperience: string;
+}
+
+export interface CvPortfolio {
+  link1: string;
```

Comment:
To zakłada tylko 1 link. Dalej dwa. Czy w specyfikacji nie było mowy o wielu?

Reply:
Zgadza się, to jeszcze Typy które były robione na początku jak nie mielismy użytkoników z BE i było wpisane trochę danych z "palca" do wyświetlania.